### DISCUSSION TITLE
How to handle the same-name files

### DISCUSSION THREAD

**Discussion by shalinjie** [Dineshkarthik/telegram_media_downloader#264] [General]
If the file name of a channel is duplicated after downloading(same name but different file content),What will it do

replace or rename?

**Comment by Dineshkarthik** [comment]
Tldr; rename

The downloader checks if there is an existing file that has been downloaded with the same name and if present it will compare the contents if they are same the new file will not be saved if it is different content then a new file with new name like `filename-copy1.pdf` will be saved.
More details can be found on this PR #30
